src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -0,0 +1,281 @@
+    Assertion.assertEquals(isElementVisible(imageInTabber), true);
+    PageObjectLogging.log("Image in tabber", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject isImageCaptionInTabberVisible() {
+    Assertion.assertEquals(isElementVisible(captionInTabber), true);
+    PageObjectLogging.log("Image caption in tabber", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject isVideoVisible() {
+    Assertion.assertEquals(isElementVisible(video), true);
+    PageObjectLogging.log("Video", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject isVideoCaptionVisible() {
+    Assertion.assertEquals(isElementVisible(videoCaption), true);
+    PageObjectLogging.log("Video caption", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject isHeroImageCentered() {
+    Assertion.assertEquals(images.get(0).getCssValue("text-align"), "center");
+    PageObjectLogging.log("Hero image", "is centered", true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject isExternalLinkLabelInURL(String name, String url) {
+    Assertion.assertStringContains(url, name);
+    PageObjectLogging.log("External links", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areUnorderedListsVisible() {
+    Assertion.assertFalse(unorderedLists.isEmpty());
+    PageObjectLogging.log("Unordered list", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areOrderedListsVisible() {
+    Assertion.assertFalse(orderedLists.isEmpty());
+    PageObjectLogging.log("Ordered list", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areHeadersVisible() {
+    Assertion.assertFalse(headers.isEmpty());
+    PageObjectLogging.log("Headers", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areLinksVisible() {
+    Assertion.assertFalse(externalLinks.isEmpty());
+    PageObjectLogging.log("Links", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areReferencesVisible() {
+    Assertion.assertFalse(references.isEmpty());
+    PageObjectLogging.log("References", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areDataLabelsVisible() {
+    Assertion.assertFalse(dataLabels.isEmpty());
+    PageObjectLogging.log("Data labels", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areDataValuesVisible() {
+    Assertion.assertFalse(dataValues.isEmpty());
+    PageObjectLogging.log("Data values", MercuryMessages.VISIBLE_MSG, true);
+
+    return this;
+  }
+
+  public PortableInfoboxObject areUnorderedListAndDataValuesMarginEqual() {
+    Assertion.assertEquals(
+        unorderedLists.get(0).getCssValue("margin"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "margin" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)